FogCreek/Glitch-Community
# Superb-guide: Fix an error caused by search results that don't exist

## Links
- https://superb-guide.glitch.me/
- https://github.com/fogcreek/glitch-community-work/issues/192

## Changes:
Check that the project exists before grabbing its permissions

## How To Test:
Search for mayfe0 (this might be time limited) and check for sentry requests in the dev tools

## Feedback I'm looking for:
Is it ok that this leaves the broken project on the page?
